dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/zorder/Zorder.scala

```diff
@@ -42,21 +46,61 @@ case class Zorder(children: Seq[Expression]) extends Expression with CodegenFall
   }
 
   @transient
-  private lazy val defaultNullValues: Seq[Any] = {
-    children.map(child => ZorderBytesUtils.defaultValue(child.dataType))
+  private lazy val defaultNullValues: JMap[Int, Array[Byte]] = {
```

Comment:
Using idx as a key seems not necessary? we can use an arrary/seq here ?

Reply:
Yes, remove the sort and update to seq.

Reply:
I think the comment of @yaooqinn is saying: `private lazy val defaultNullValues: Array[Array[Byte]]`, we don't need map here.

Reply:
Sorry, I misread before, using an array here is ok, reverted it.